mvnw.cmd

```diff
@@ -118,7 +118,7 @@ for /F "usebackq delims=" %%a in ("%MAVEN_PROJECTBASEDIR%\.mvn\jvm.config") do s
 
 SET MAVEN_JAVA_EXE="%JAVA_HOME%\bin\java.exe"
 
-set WRAPPER_JAR=""%MAVEN_PROJECTBASEDIR%\.mvn\wrapper\maven-wrapper.jar""
+set WRAPPER_JAR="%MAVEN_PROJECTBASEDIR%\.mvn\wrapper\maven-wrapper.jar"
```

Comment:
Without this change, I'm seeing the following error on Windows when running `mvnw.cmd` in a directory name that contains spaces:

```
Error: Could not find or load main class With Spaces\.mvn\wrapper\maven-wrapper.jar
```

:+1: